package com.sky.service;

import com.sky.dto.EmployeeDTO;
import com.sky.dto.EmployeeLoginDTO;
import com.sky.dto.EmployeePageQueryDTO;
import com.sky.entity.Employee;
import com.sky.result.PageResult;
import com.sky.result.Result;
import org.springframework.web.bind.annotation.PathVariable;

public interface EmployeeService {

    /**
     * 员工登录
     * @param employeeLoginDTO
     * @return
     */
    Employee login(EmployeeLoginDTO employeeLoginDTO);

    public void register(EmployeeDTO employeedto);

    public PageResult pageQuery(EmployeePageQueryDTO employeePageQueryDTO);

    public void enableOrDisable(Integer status, Long id);

}
